src/components/standalone/InputLayout.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" setup>
+defineProps({
+  title: String,
+  description: String
+})
+
+defineOptions({
+  inheritAttrs: false
+})
+</script>
+
+<template>
+  <div class="flex flex-col md:flex-row">
+    <div class="md:w-5/12 md:pr-6">
+      <slot name="title">
+        <h2 class="font-medium text-xl mb-2">{{ title }}</h2>
+      </slot>
+      <slot name="description">
+        <p class="font-normal text-sm text-gray-400 mb-8">{{ description }}</p>
+      </slot>
+    </div>
+    <div class="md:w-7/12" v-bind="$attrs">
```

Comment:
```suggestion
    <div class="md:w-7/12">
```
Let's keep the default policy for component attributes